.github/workflows/main.yml

```diff
@@ -26,8 +26,8 @@ jobs:
             os: ubuntu-latest
             test_config: "NEP29"
           # Operational compliance settings
-          - python-version: "3.6.8"
-            numpy_ver: "1.19.5"
+          - python-version: "3.9"
```

Comment:
Since the most recent pysat release kept 3.6.8 as the "operational" version, for consistency,  pysatPenumbra packages should maintain the same testing version.

Reply:
No, this one is just leading the charge.

Reply:
We need to update the libraries before the core pysat can be updated.